Snowflake.Data.Tests/SFBindTestIT.cs

```diff
@@ -686,6 +687,83 @@ public void testPutArrayBind()
                 conn.Close();
             }
         }
+        
+        [Test]
+        public void testPutArrayBindWorkDespiteOfProblematicTypeNameHandlingValue()
+        {
+            JsonConvert.DefaultSettings = () => new JsonSerializerSettings {
+                TypeNameHandling = TypeNameHandling.Auto
+            };
+            
+            
+            using (IDbConnection conn = new SnowflakeDbConnection())
+            {
+                conn.ConnectionString = ConnectionString;
+                conn.Open();
+
+                using (IDbCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = "create or replace table testPutArrayBind(cola REAL, colb TEXT, colc NUMBER(38,0))";
+                    int count = cmd.ExecuteNonQuery();
+                    Assert.AreEqual(0, count);
+
+                    string insertCommand = "insert into testPutArrayBind values (?, ?, ?)";

```

Comment:
shouldn't we use `var` where type can be determined by compiler? I won't refer to that in lines below anymore

Reply:
ok